ciao-controller/instance.go

```diff
@@ -166,23 +191,71 @@ func getStorage(c *controller, wl *types.Workload, tenant string) (payloads.Stor
 			return payloads.StorageResources{}, err
 		}
 
+		// don't you need to add support for indicating whether
```

Comment:
I expect the original failure was due to https://github.com/01org/ciao/pull/621 which was present in this PR till I rebased just now. The issue of continuing to issue the start request despite the failure has been addressed in a separate commit.